build.gradle

```diff
@@ -89,11 +89,12 @@ jenkinsPlugin {
 }
 
 dependencies {
-    api platform("io.jenkins.tools.bom:bom-2.289.x:${jenkinsBomVersion}")
-    implementation 'org.jenkins-ci.plugins:credentials'
-    implementation 'org.jenkins-ci.plugins:structs'
+    api platform("io.jenkins.tools.bom:bom-2.440.x:${jenkinsBomVersion}")
+    implementation  'org.jenkins-ci.plugins:credentials'
+    implementation  'org.jenkins-ci.plugins:structs'
     implementation 'org.jodd:jodd-http:5.0.1'
     implementation "org.codehaus.groovy:groovy-all:${groovyVersion}"
+    implementation 'com.github.jnr:jnr-ffi:2.2.17'
```

Comment:
Is this needed?  Where is this library being used?  There was an unused import in `CTMExecute.java` but I see no other usages in the code.